modules/react/skeleton/lib/skeleton.tsx

```diff
@@ -1,10 +1,18 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
 import {keyframes} from '@emotion/core';
-import canvas from '@workday/canvas-kit-react/tokens';
-import {accessibleHide} from '@workday/canvas-kit-react/common';
 
-export interface SkeletonProps {
+import {
+  accessibleHide,
+  createComponent,
+  styled,
+  StyledType,
+} from '@workday/canvas-kit-react/common';
+
+import SkeletonHeader from './parts/skeletonHeader';
+import SkeletonShape from './parts/skeletonShape';
+import SkeletonText from './parts/skeletonText';
+
+interface SkeletonProps {
   /**
    * For accessibility reasons, `aria-label` is transformed into a text representation
    * (visually hidden, but announced by screen readers) of the loader.
```

Comment:
I did not get to look into "We anticipate that this will change in a future major version."